src/Evolution/DiscontinuousGalerkin/Limiters/WenoOscillationIndicator.cpp

```diff
@@ -79,9 +80,9 @@ double compute_sum_of_legendre_derivs(
 }
 
 // Compute the indicator matrix, similar to that of Dumbser2007 Eq. 25, but
-// adapted for use on square/cube grids.
+// adapted for use on square/cube grids. A Legendre basis is assumed.
```

Comment:
Do I understand correctly that no functionality actually changed in this commit, but you just relaxed asserts? So everything essentially already worked on Gauss grid before?

Reply:
That's right! The previous code would correctly compute the matrix for either Gauss or GaussLobatto grids. But the caching was more restrictive, and would work with only of the two quadratures. This commit reworks the caching to support both at the same time (to make testing easier), adds tests, and relaxes asserts. I'll expand the commit message :)